docs/getting_started/modin_vs_dask_vs_koalas.rst

```diff
@@ -0,0 +1,91 @@
+* indexing & ordering semantics, unlike DaskDF and Koalas that deviate from these semantics
+* eager execution, unlike DaskDF and Koalas that provide lazy execution
+
+As a result, Modin's coverage is `more than 90% <https://github.com/modin-project/modin#pandas-api-coverage>`_ of the pandas API, while DaskDF and Koalas' coverage is about 55%. 
+
+.. figure:: ../img/api_coverage_comparison.svg
+   :align: center
+   :alt: Percentage coverage of the pandas API after deduplication
+
+For more technical details please see our VLDB 2022 paper, referenced `here <https://people.eecs.berkeley.edu/~totemtang/paper/Modin.pdf>`_. 
+
+Brief Overview of DaskDF and Koalas
+-----------------------------------
+
+Dask's `DataFrame <https://docs.dask.org/en/stable/dataframe.html>`_ (DaskDF) is effectively a meta-frame, partitioning and scheduling many smaller ``pandas.DataFrame`` objects. Users construct a task graph of dataframe computation step by step and then trigger computation using the ``compute`` function.
+
+
+Spark's `Koalas <https://koalas.readthedocs.io/en/latest/>`_ provides the pandas API on Spark, leveraging the preexisting Spark SQL optimizer to execute select pandas commands. Like DaskDF, Koalas also employs lazy computation, only triggering computation when the user requests to see the results.
+
+
+
+Partitioning and Parallelization
+--------------------------------
+**DaskDF and Koalas only support row-oriented partitioning and parallelism.** This approach is analogous to relational databases. The dataframe is conceptually broken down into horizontal partitions along rows, where each partition is independently processed if possible. When DaskDF or Koalas are required to perform column-parallel operations that to be done on columns independently (e.g., dropping columns with null values via ``dropna`` on the column ``axis``), they either perform very poorly with no parallelism or do not support that operation.
+
+
+**Modin supports both row, column, and block-oriented partitioning and parallelism**. That is, the dataframe can be conceptually broken down as groups of rows, groups of columns, or both groups of rows and groups of columns (effectively a block or sub-matrix). Modin will transparently reshape the partitioning as necessary for the corresponding operation, based on whether the operation is row-parallel, column-parallel, or cell-parallel (independently applied to each unit cell). This allows Modin to support more of the pandas API and do so efficiently. Due to the finer-grained control over the partitioning, Modin can support a number of operations that are very challenging to parallelize in row-oriented systems (e.g., ``transpose``, ``median``, ``quantile``). This flexibility in partitioning also gives Modin tremendous power to implement efficient straggler mitigation and improve utilization over the entire cluster.
+
+
+API Coverage
```

Comment:
Wondering if we should have the "API Coverage" section before the "Partitioning and Parallelization" section. 
Right now it feels like the partitioning is not well motivated, users don't care about partitioning for its own sake but for the capabilities, they afford (note that this page also sits in the "Getting Started" guide). If we put API first, we can say here is the higher API coverage, and how do we do this through partitioning. Perhaps it is worth adding a sentence at the top of the "Partitioning and Parallelization" section that says something like *"Modin, DaskDF, Koalas are all examples of parallel dataframe system. Parallelism is achieved through partitioning a large dataframe into smaller units that can be operated on. As a result, the partitioning scheme chosen by the library dictates what pandas functions can or can not be supported."*

Reply:
It is hard to understand why the API coverage is higher without having described row/column/cell-based partitioning first, so I am more in favor of having that be first. 

That said, I like your idea of adding a sentence at the start of partitioning for why we're talking about it, but small rewording/fixing typos: _Modin, DaskDF, Koalas are all examples of parallel dataframe systems. Parallelism is achieved by partitioning a large dataframe into smaller ones that can be operated on in parallel. As a result, the partitioning scheme chosen by the system dictates the pandas functions that can or can not be supported.__

Reply:
Added the header paragraph to Partitioning section